src/Compilers/Shared/BuildServerConnection.cs

```diff
@@ -472,7 +472,7 @@ private static bool CheckPipeConnectionOwnership(NamedPipeClientStream pipeStrea
             }
         }
 
-#if NETSTANDARD1_3
+#if NET46
```

Comment:
I would have expected far more of these #if's. Did you search to make sure there are no other mentions of NETSTANDARD1_3?

Reply:
Same. There just weren't that many in the core compiler code base.